picard/formats/mp4.py

```diff
@@ -117,6 +117,14 @@ def _load(self, filename):
         if tags is None:
             file.add_tags()
 
+        if not tags:
+            # MPEG-4 file in an mov container is unsupported and is one of
+            # the formats which returns a positive score in guess_format method
+            # If we are unable to add tags, it is not a supported mp4 file
+            # remove it and log an error
+            log.error("Unable to load tags for file {}".format(self))
+            self.remove()
+            return {}
```

Comment:
I would just return `Metadata()`. Assuming parent class is a dict looks weird to me.